ddl/ddl_worker.go

```diff
@@ -362,6 +401,7 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx, shouldCleanJobs bool) error {
 			// No job now, return and retry getting later.
 			return nil
 		}
+		w.waitDependencyJobFinished(job, &waitDependencyJobCnt)
```

Comment:
If its dependencyJob is not done yet, it would return at line 357. So why we need to wait here?

Reply:
line 357 return is in a `txn` func, not return in `handleDDLJobQueue func`

Reply:
As @crazycs520 said. And if put it in line357, we need wait for 200ms. I am afraid this `txn` is easy to conflict. So I put it here.